### DISCUSSION TITLE
How can I remove the guide tab from the menu please? And how can I change the order of the tabs?

### DISCUSSION THREAD

**Discussion by yanghui-wang** [academicpages/academicpages.github.io#2047] [Q&A] [answered]
How can I remove the guide tab from the menu please? And how can I change the order of the tabs?

**Comment by rjzupkoii** [comment, marked as answer]
The guide tab is determined by the contents of `_data/navigation.yml`. Removing the following lines:

```yml
  - title: "Guide"
    url: /markdown/
```

Will remove the guide from the the menu. Additionally, changing the order of the entries determines the order that the guide tab entries appears in.
